addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.js

```diff
@@ -78,22 +82,47 @@ class ComposerTextInput extends Component {
      * Saves the composer text input state in store
      */
     saveStateInStore() {
-        this.composer.saveTextInput({
+        this.composer.update({
             textInputContent: this.getContent(),
```

Comment:
Ok for ```getContent``` being private but I'm not sure to clearly get the second part of your comment

Reply:
From what I saw when I posted the comment, `getContent` is called from the outside, so it can only be made private after the other component using it is not using it anymore, and I gave hints on how to probably do it.